### PR TITLE
Remove SSL certificate DN normalization workaround

### PR DESCRIPTION
**Pull request by ekohl**
This was fixed in serverspec 2.41.6 and no longer needs a workaround.

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: spec/acceptance/foreman_proxy_spec.rb [hunk 1]
```diff
@@ -28,8 +28,8 @@
       it { should be_certificate }
       it { should be_valid }
       it { should have_purpose 'server' }
-      include_examples 'certificate issuer', "C = US, ST = North Carolina, L = Raleigh, O = Katello, OU = SomeOrgUnit, CN = #{FQDN}"
-      include_examples 'certificate subject', "C = US, ST = North Carolina, O = FOREMAN, OU = SMART_PROXY, CN = #{FQDN}"
+      its(:issuer) { "C = US, ST = North Carolina, L = Raleigh, O = Katello, OU = SomeOrgUnit, CN = #{FQDN}") }
+      its(:subject) { "C = US, ST = North Carolina, O = FOREMAN, OU = SMART_PROXY, CN = #{FQDN}") }
```
Comment (ehelms) [inline spec/acceptance/foreman_proxy_spec.rb:32]: Missing the `should eq` part

**Comment by ehelms** [comment]
Check all the files, there are multiple places missing the `Missing the `should eq` part`

**Comment by ekohl** [comment]
Rebased since there were some cases added in other PRs.

**Comment by ekohl** [comment]
Now it's finally green.

**Pull request merged**